server/test/unit/vitalsTest/checkForInternalProblemsTest.js

```diff
@@ -0,0 +1,215 @@
+    })
+
+    it('should not send any alerts to Sentry', () => {
+      expect(helpers.logSentry).to.not.have.been.called
+    })
+
+    it('should not log any errors', () => {
+      expect(helpers.log).to.not.be.called
+    })
+  })
+
+  describe('duration alerts should not be counted, even if duration alerts exceed maxStillnessAlerts in 24 hours for a location', () => {
+    beforeEach(async () => {
+      // Insert a single location and maxStillnessAlerts sessions + 1
+      const client = await factories.clientDBFactory(db, { isActive: true })
+      this.testLocation = await locationDBFactory(db, {
+        clientId: client.id,
+        isActive: true,
+      })
+      for (let i = 1; i <= maxStillnessAlerts + 1; i++) {
+        this[`session${i}`] = await sessionDBFactory(db, {
+          locationid: this.testLocation.locationid,
+          alertType: ALERT_TYPE.SENSOR_DURATION,
+          incidentCategory: ALERT_TYPE.SENSOR_DURATION,
+        })
+        await helpers.sleep(1) // sleep to make sure that these have created_at values at least one millisecond apart
+      }
+      for (let i = maxStillnessAlerts + 2; i <= maxStillnessAlerts + 3; i++) {
+        this[`session${i}`] = await sessionDBFactory(db, {
+          locationid: this.testLocation.locationid,
+          alertType: ALERT_TYPE.SENSOR_STILLNESS,
+          incidentCategory: ALERT_TYPE.SENSOR_STILLNESS,
+        })
+        await helpers.sleep(1) // sleep to make sure that these have created_at values at least one millisecond apart
+      }
+      await vitals.checkForInternalProblems()
+    })
+
+    it('should not send any alerts to Sentry', () => {
+      expect(helpers.logSentry).to.not.have.been.called
+    })
+
+    it('should not log any errors', () => {
+      expect(helpers.log).to.not.be.called
+    })
+  })
+
+  describe('should log to Sentry after finding more than maxStillnessAlerts have occurred in 24 hours from a single location', () => {
+    beforeEach(async () => {
+      // Insert a single location and maxStillnessAlerts sessions + 1
+      const client = await factories.clientDBFactory(db, { isActive: true })
+      this.testLocation = await locationDBFactory(db, {
+        clientId: client.id,
+        isActive: true,
+      })
+
+      this.incidentCategory = ALERT_TYPE.SENSOR_STILLNESS
+      this.alertType = ALERT_TYPE.SENSOR_STILLNESS
+      for (let i = 1; i <= maxStillnessAlerts + 1; i++) {
+        this[`session${i}`] = await sessionDBFactory(db, {
+          locationid: this.testLocation.locationid,
+          alertType: this.alertType,
+          incidentCategory: this.incidentCategory,
+        })
+        await helpers.sleep(1) // sleep to make sure that these have created_at values at least one millisecond apart
+      }
+      await vitals.checkForInternalProblems()
+    })
+
+    it('should send one max stillness alert to Sentry', () => {
+      expect(helpers.logSentry).to.have.been.calledOnce
+    })
```

Comment:
Is this test necessary since you have the one right after it testing `it('should send one max stillness alert specific to the location that has exceeded the maxStillnessAlerts in 24 hours to Sentry'`?

Reply:
That's a good point! I think I was thinking that if the second test fails, then I at least have this test to indicate that some type of message was received. But looking at the error messages I get, I think I can take it out!